File: python/7 kyu/211.py
# https://www.codewars.com/kata/5bbd279c8f8bbd5ee500000f/train/python

# Passed

def find_screen_height(width, ratio):
    height_and_width = [int(num) for num in ratio.split(":")[::-1]]
    height = width * height_and_width[0] / height_and_width[1] # h = w * (h/w)
    height = int(height)
    screen_dimensions = f"{width}x{height}"
    return screen_dimensions

output = find_screen_height(1920, "16:9")
print(output)

File: nodejs/7 kyu/211.js
// https://www.codewars.com/kata/56aed5db9d5cb55de000001c/train/javascript

// Passed

function twoCount(n) {
  let result = 0;
  while (n % 2 === 0) {
    result++;
    n /= 2;
  }

  return result;
}

const output = twoCount(17280);
console.log(output);